dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -358,9 +358,19 @@ public GitHubJsonStore getGitHubJsonStore(String store) {
 
     public void createPullReq(GHRepository origRepo,
```

Comment:
So we should have some javadocs on this method, especially since the rate limiter parameter is nullable.